package.json

```diff
@@ -41,15 +41,16 @@
   "bugs": "https://github.com/html-next/ember-gestures/issues",
   "dependencies": {
     "animation-frame": "~0.2.4",
-    "broccoli-funnel": "^2.0.1",
-    "broccoli-merge-trees": "^2.0.0",
-    "broccoli-stew": "^1.5.0",
+    "broccoli-funnel": "^2.0.2",
+    "broccoli-merge-trees": "^3.0.2",
+    "ember-auto-import": "^1.3.0",
```

Comment:
how does ember-auto-import come into play here?

Reply:
It doesn't I just left it in after an experiment, removed it today.